src/Pages/About/About.js

```diff
@@ -1,27 +1,78 @@
 import React from "react";
+import "./About.css";
+
+import hugeLogo from "../../images/hugeLogo.png";
 
 export default function About() {
+    let podCastData = [
+        {
+            name: "Spotify",
+            link: "https://open.spotify.com/show/2dTRwdB4bHYsCI2CN9OV2m",
+        },
+        { name: "Libsyn", link: "https://sites.libsyn.com/447303/" },
```

Comment:
this is the biggest nitpick yet and your linter probably did it automatically but it would be nice if all the objects were shaped the same.
Feel free to tell me to gfy on this one though lol

Reply:
every time I save it differently the linter puts it back

Reply:
I figured so much. No worries then!